src/components/Footer.tsx

```diff
@@ -3,15 +3,25 @@ import Link from "next/link"
 
 function Footer() {
   return (
-    <footer className="flex justify-center gap-4 p-5">
+    <footer className="grid grid-cols-2 place-items-center gap-4 p-5">
       {/* Testing self-end with this div */}
+      <div>
+        <p className="text-3xl">JDELTA Properties</p>
+      </div>
+      <div>
+        <p className="text-2xl">Address</p>
+        <p className="text-sm">1111 JDELTA Drive</p>
+        <p className="text-sm">Shreveport, Louisiana</p>
```

Comment:
Ditch address and any "real" data for now. Throw in fantasy wonderland or something